tests/expose_test.go

```diff
@@ -663,10 +656,8 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 				// This TC also covers:
 				// [test_id:1795] Exposed VM (as a service) can be reconnected multiple times.
-				By("Getting back the cluster IP given for the service")
-				svc, err := virtClient.CoreV1().Services(vm.Namespace).Get(serviceName, k8smetav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				serviceIP := svc.Spec.ClusterIP
+				serviceIP, err := getServiceIP(virtClient, vm.GetNamespace(), serviceName)
```

Comment:
Can we pass the service object instead? Then there will be no need to fetch the namespace from another object, it is confusing.

Reply:
Removed these functions.